src/components/Navbar/Navbar.spec.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
I really liked that you tested even snapshots, great, but would add value also if you test the search input logic